odoo/addons/base/models/ir_model.py

```diff
@@ -1491,6 +1492,48 @@ def group_names_with_access(self, model_name, access_mode):
         """ % access_mode, [model_name])
         return [('%s/%s' % x) if x[0] else x[1] for x in self._cr.fetchall()]
 
+    @tools.ormcache('self.env.uid', 'self.env.su')
+    def _acl_map(self):
+        """ Return a mapping of model access rights as ``access[model_name][model]``. """
+        if self.env.su:
+            all_true = {'read': True, 'write': True, 'create': True, 'unlink': True}
+            return collections.defaultdict(lambda: all_true)
+
+        self.flush(self._fields)
+        # note: need a subquery because it's possible to get models with IMA
+        #       with only groups the user doesn't have, in which case we
+        #       generate a bunch of rows with gu.uid != self.env.uid (satisfying
+        #       left joins) which get filtered out afterwards, ending up with no
+        #       row for the model...
+        self._cr.execute("""
+               SELECT model, true, true, true, true FROM ir_model WHERE transient
+        UNION ALL
```

Comment:
Beware: in saas-13.2 we removed this special case for transient models #43306.
So you will have to remove those lines in saas-13.2.